bql/semantic/expression.go

```diff
@@ -441,8 +425,8 @@ func internalNewEvaluator(ce []ConsumedElement) (Evaluator, []ConsumedElement, e
 			return e, res, nil
 		}
 
-		if bndTkn.Type == lexer.ItemNode {
-			e, err := NewEvaluationExpressionForNodeLiteral(op, tkn.Text, bndTkn.Text)
+		if bndTkn.Type == lexer.ItemLiteral {
```

Comment:
Out of curiosity, why change the ordering of the blocks?

Reply:
As I was navigation the code it seemed more natural the ordering `comparisonForLiteral` -> `comparisonForNodeLiteral` -> `comparisonForPredicateLiteral` (in a following PR). So I changed the ordering here inside `internalNewEvaluator` too to be coherent with that. I can also change back to the previous ordering if you prefer.